lib/heroku/helpers/heroku_postgresql.rb

```diff
@@ -109,6 +109,11 @@ def hpg_databases
       @hpg_databases = Hash[ pairs ]
 
       # TODO: don't bother doing this if DATABASE_URL is already present in hash!
+      # CB: this line is falsing from cb-rails 3 because i do have a DATABASE_URL attachment
+      # so it doesn't resolve to real
+      # Looks like the API is returning the DATABASE_URL as a separate attachment
+      # than the color attachment
+      # the UUIDs are different, but it's the same ["resource"]["id"]
```

Comment:
This is by design. In the past these attachments were hidden ("implicit attachments") but continuing to do so doesn't make sense in light of shareabale add-ons. i.e. DATABASE shouldn't be treated specially.